GoRest.Api.Tests/Users/Update/PUTUpdateUserNegativeTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using GoRest.Api.Client.Client;
+using GoRest.Api.Client.Client.Models;
+using GoRest.Api.Client.Client.Interfaces.Controllers;
+using NUnit.Framework;
+using System;
+using GoRest.Api.Client.Client.Models.UsersApi;
+using GoRest.Api.Client.Client.Builder;
+
+namespace GoRest.Api.Tests.Users
+{
+    [TestFixture]
+    public class PutUpdateUserNegativeTests
+    {
+        [Test]
+        public async Task VerifyUserIsNotUpdatedWithAlreadyPresentInDBEmail()
+        {
+            // Arrange
+            var responseCreateFirstUser = await GoRestClient.For<IUsersApi>().CreateUser(new CreateUserBuilder().Build());
+            responseCreateFirstUser.Code.Should().Be(HttpStatusCode.Created);
+
+            var responseCreateSecondUser = await GoRestClient.For<IUsersApi>().CreateUser(new CreateUserBuilder().Build());      
+            var userId = responseCreateSecondUser.Data.Id.ToString();
+
+            var userModelUpdateSecondUserWithEmilOfFirstUser = new GeneralResponseModel()
```

Comment:
create a builder for the update operation. and use it

Reply:
done